src/google/composer.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "scheb/2fa-google",
+    "type": "library",
+    "description": "Provides two-factor authentication using Google Authenticator",
+    "keywords": ["two-factor", "two-step", "authentication", "security", "symfony"],
+    "homepage": "https://github.com/scheb/two-factor-bundle",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Christian Scheb",
+            "email": "me@christianscheb.de"
+        }
+    ],
+    "require": {
+        "scheb/2fa-bundle": "self.version",
```

Comment:
Shouldn't that be `^5.0`? Otherwise you'd be forced to always update all 2fa-* packages.

I see how this could make sense for the "core" packages (like backup codes and trusted devices), but for the authentication providers maybe not so much.

On the other side, I see how this ensures that the combination is packages with the same version is guaranteed to be compatible, since they're originating from the same main repository.